osu.Game/Overlays/NotificationOverlay.cs

```diff
@@ -84,9 +112,13 @@ private void notificationClosed()
                 State = Visibility.Hidden;
         }
 
+        private readonly Scheduler postScheduler = new Scheduler();
+
+        private bool processingPosts = true;

```

Comment:
Can just use `postScheduler.Enabled` instead of local bool.

Reply:
used to add the delay (without affecting the `BindableBool`'s state

Reply:
I mean the Scheduler's Enabled bool, not the bindable.

Reply:
only available in `ThreadedScheduler` :(